tests/features/7/https.feature

```diff
@@ -1,6 +1,11 @@
 @jboss-eap-7 @jboss-eap-7-tech-preview
 Feature: Check HTTPS configuration
 
+  # We are not able to test the following fail-fast error conditions
+  #
+  # * No undertow subsystem - it is too 'ingrained' to remove without changing a load of stuff
+  # * No undertow servers - we cannot unset the default-server attribute so we cannot remove the default-server
```

Comment:
-e GALLEON_PROVISION_LAYERS=core-server with s2i, this would provision a server without undertow.

Reply:
Additionally to test not only missing undertow but also a missing server, cannot we just copy a manually crafted openshift-standalone.xml so the behave tests use it instead of the default one?

Reply:
For missing subsystem/server/hosts I think we can just start from the no undertow subsystem case and then use the following simple scripts to add the following steps
* add undertow
* add a server
* add a host